l1-contracts/contracts/state-transition/TestnetVerifier.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import {Verifier} from "./Verifier.sol";
+
+/// @author Matter Labs
+/// @custom:security-contact security@matterlabs.dev
+/// @notice Modified version of the main verifier contract for the testnet environment
+/// @dev This contract is used to skip the zkp verification for the testnet environment.
+/// If the proof is not empty, it will verify it using the main verifier contract,
+/// otherwise, it will skip the verification.
+contract TestnetVerifier is Verifier {
+    constructor() {
+        assert(block.chainid != 1);
+    }
+
+    function verify(
```

Comment:
please add doc comments

Reply:
Added in b3d48b86f6f198b7952e47a9f26638f9f8118616